README.md

```diff
@@ -296,6 +296,14 @@ service = client.get_service "guestbook", 'development'
 Note - Kubernetes doesn't work with the uid, but rather with the 'name' property.
 Querying with uid causes 404.
 
+#### Get the raw response for an entity or entities
```

Comment:
could simplify to `Getting raw responses`
and maybe add "To speed up processing or reduce memory consumption return raw JSON."